core/modules/css/PartTableParams.h

```diff
@@ -100,6 +104,16 @@ struct PartTableParams {
         if (not dirColName.empty()) res.push_back(dirColName);
         return res;
     }
+
+    /// Returns true if table should be locked in memory for shared scan
+    bool lockInMemory() const {
+        return lockInMem;
+    }
+
+    /// Returns the prefered speed of shared scan for the table
+    int getScanSpeed() const {
```

Comment:
These two new methods are not needed, this is a struct and members can be accessed directly.